src/style.css

```diff
@@ -0,0 +1,43 @@
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  width: 15px;
+  height: 15px;
+  padding: 0 10px;
+  background-image: url(../src/images/star-active.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+}
+
+.stars--0 .stars__star:nth-child(1),
+.stars--0 .stars__star:nth-child(2),
+.stars--0 .stars__star:nth-child(3),
+.stars--0 .stars__star:nth-child(4),
+.stars--0 .stars__star:nth-child(5) {
+  background-image: url(../src/images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(2),
+.stars--1 .stars__star:nth-child(3),
+.stars--1 .stars__star:nth-child(4),
+.stars--1 .stars__star:nth-child(5) {
+  background-image: url(../src/images/star.svg);
+}
+
+.stars--2 .stars__star:nth-child(3),
+.stars--2 .stars__star:nth-child(4),
+.stars--2 .stars__star:nth-child(5) {
+  background-image: url(../src/images/star.svg);
+}
+
+.stars--3 .stars__star:nth-child(4),
+.stars--3 .stars__star:nth-child(5) {
+  background-image: url(../src/images/star.svg);
+}
+
+.stars--4 .stars__star:nth-child(5) {
+  background-image: url(..//src/images/star.svg);
+}
```

Comment:
2'nd point in checklist, also you don't need that many selectors

Reply:
Why?

Reply:
because you can use `nth-child` pseudo class instead